doc/input-output-reference/src/overview/group-radiative-convective-units.tex

```diff
@@ -571,15 +571,15 @@ \subsubsection{Inputs}\label{inputs-3-028-1}
 
 \paragraph{Field: Rated Water Mass Flow Rate}\label{field-rated-water-mass-flow-rate}
 
-This field is the rated standard water flow rate in kg/s that was used to obtain the rated capacity under rated conditions.  This value along with the other rated conditions is used to calculate the overall heat transfer coefficient (U-Value) that will be used to characterize the heat transfer between the water being circulated through the panel and the air in the space.  The default value is 0.063kg/s (1 gpm).  If it is blank or zero, the default value is assumed.
+This field is the rated standard water flow rate in kg/s that was used to obtain the design cooling capacity under rated temperature conditions.  This value along with the rated space and water inlet temperatures as well as the cooling capacity (whether autosized or entered by the user) to calculate the overall heat transfer coefficient (U-Value) that will be used to characterize the heat transfer between the water being circulated through the panel and the air in the space.  Users should note that if this value is too low in comparison to the cooling design capacity that the program will produce an error message and the user must adjust one of these two parameters or the rated temperature values.  The default value is 0.063kg/s (1 gpm).  If it is blank or zero, the default value is assumed.
```

Comment:
@jasondegraw I agree...I thought I had commented on this earlier and it is not showing up here now.  Anyway, yes, I will correct this as soon as possible.